src/main/java/com/jcabi/github/RtUser.java

```diff
@@ -138,6 +141,29 @@ public UserEmails emails() {
         return new RtUserEmails(this.ghub.entry());
     }
 
+    /**
+     * Returns all notifications of a user.
+     * @see <a href="https://developer.github.com/v3/activity/notifications/#list-your-notifications">List your notifications</a>
+     * @return Returns all notifications for this user.
+     * @todo #913:30min Implement method RtUser.notifications.
```

Comment:
Remove javadoc and add a single todo for `notification` and `markAsRead` methods in class description